mfs/mfs_test.go

```diff
@@ -552,7 +551,8 @@ func actorMakeFile(d *Directory) error {
 		return err
 	}
 
-	r := io.LimitReader(randbo.New(), int64(77*rand.Intn(123)))
+	rread := rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
if we already import from there, then sure, lets use it. Otherwise i'd like to avoid another import of that package. I want it to die.